Tests/GRPCInProcessTransportTests/InProcessServerTransportTests.swift

```diff
@@ -41,7 +41,7 @@ final class InProcessServerTransportTests: XCTestCase {
 
     try await withThrowingTaskGroup(of: Void.self) { group in
       group.addTask {
-        try await transport.listen { stream in
```

Comment:
Aren't we getting a warning because we're not using `context` here? Should we `_` it, or should we assert the contained method descriptor?